src/test/java/edu/hm/hafner/analysis/ModuleDetectorTest.java

```diff
@@ -140,26 +334,35 @@ void shouldIdentifyModuleByReadingAntProjectFile() {
         });
         ModuleDetector detector = new ModuleDetector(ROOT, factory);
 
-        assertThat(detector.guessModuleName(PREFIX + (PATH_PREFIX_ANT + "/something.txt")))
+        assertThat(detector.guessModuleName(PREFIX + (PATH_PREFIX_ANT + "something.txt")))
                 .isEqualTo(EXPECTED_ANT_MODULE);
-        assertThat(detector.guessModuleName(PREFIX + (PATH_PREFIX_ANT + "/in/between/something.txt")))
+        assertThat(detector.guessModuleName(PREFIX + (PATH_PREFIX_ANT + "in/between/something.txt")))
                 .isEqualTo(EXPECTED_ANT_MODULE);
-        assertThat(detector.guessModuleName(PREFIX + "/path/to/something.txt"))
+        assertThat(detector.guessModuleName(PREFIX + "path/to/something.txt"))
                 .isEqualTo(StringUtils.EMPTY);
     }
 
     @Test
     void shouldIgnoreExceptionsDuringParsing() {
```

Comment:
It looks like this test wasn't working properly, because it was never trying to open() any of the files, so the stub throw was never actually throwing.

FIxed it by returning a file set containing each type that would try to open() and parse the files, in order to trigger the exceptions.

Reply:
Thanks for spotting this!